2.7/s2i/bin/run

```diff
@@ -71,14 +75,12 @@ if should_migrate; then
 fi
 
 if is_gunicorn_installed; then
-  if [[ -z "$APP_MODULE" ]]; then
-    # Look only in the directory specified by APP_HOME, or the current directory
-    # replace all "/" with ".", remove leading "." and ".py" suffix
-    APP_MODULE=$(find $APP_HOME -maxdepth 1 -type f -name 'wsgi.py' | sed 's:/:.:g;s:^\.\+::;s:\.py$::')
-  fi
-
-  if [[ -z "$APP_MODULE" && -f setup.py ]]; then
-    APP_MODULE="$(python setup.py --name)"
+  setup_py=$(find "$HOME" -maxdepth 2 -type f -name 'setup.py' -print -quit)
```

Comment:
Why `$HOME` instead of `$APP_HOME`?

Reply:
Right, sources are copied to the $HOME directory so setup.py should be there. The `-maxdepth 2` is just for safety I presume?

Reply:
I realize it's just a detail, but for readability's sake:
Since we're using $HOME in this place, could we also use it when we move the scripts there?

https://github.com/sclorg/s2i-python-container/blob/0b8fa29097d5e179dde6e2267c44f2e1d5f6fa5b/2.7/s2i/bin/assemble#L38

Right now it says `mv /tmp/src/* ./`, which is quite cryptic unless you know where you're currently located, using `$HOME/` would make it much clearer for anyone trying to understand it.

Reply:
Makes sense to me. Changed in the latest commit.